plotly/tools.py

```diff
@@ -28,6 +28,10 @@
                          'rgb(227, 119, 194)', 'rgb(127, 127, 127)',
                          'rgb(188, 189, 34)', 'rgb(23, 190, 207)']
 
+#color constants for violin plot
+fillcolor = '#1f77b4'
```

Comment:
I could have been clearer on this. Let's call this something like:

`DEFAULT_FILL_COLOR = '#1f77b4'`

Constants should be in `CAPS`.